tensorflow/python/framework/ops.py

```diff
@@ -265,7 +265,9 @@ class Tensor(_TensorLike):
       # Unary.
       "__invert__",
       "__neg__",
-      "__abs__"
+      "__abs__",
+      "__matmul__",
```

Comment:
This is needed to allow `_OverrideBinaryOperatorHelper(matmul, "matmul")` in `math_ops.py`.